### ISSUE TITLE
scripts to generate the circuits

### ISSUE THREAD

**Issue by alexandrupaler** [glassnotes/FT_qRAM_Circuits#2]
do you have scripts to generate parameterised versions of the circuits you analyse?

**Comment by glassnotes** [comment]
Do you mean parameterized in terms of the n, k, q in the manuscript? I just use for loops, there are some examples in the file scripts/generate_paper_data.py.

**Comment by alexandrupaler** [comment]
Not really. I mean parameterised in order to generate a gate list for the circuit. Something that can be, for example, executed with ProjectQ.

**Comment by glassnotes** [comment]
Ah, I see! This isn't implemented yet, but is something that I am planning to have done in the future (by a student). The idea was to use Q#, but it would probably be good to make something more general as well.

**Comment by alexandrupaler** [comment]
Thank you. Q# would be good, because I am also interested in porting some
stuff to it. It would be great if you could share the scripts, once you
have them.

**Comment by glassnotes** [comment]
For sure, they will all be made available open source! Unfortunately though the time horizon is a little far ahead (earliest potential start for a student would be May 2020).
